plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -157,6 +227,95 @@ export const PricingSection: React.FC = () => {
 							) }
 						/>
 					</BaseControl>
+
+					<ToggleControl
+						label={
+							<>
+								{ __( 'Schedule sale', 'woocommerce' ) }
+								<Tooltip
+									text={ interpolateComponents( {
+										mixedString: __(
+											'The sale will start at the beginning of the "From" date ({{startTime/}}) and expire at the end of the "To" date ({{endTime/}}). {{moreLink/}}',
+											'woocommerce'
+										),
+										components: {
+											startTime: (
+												<span>
+													{ formatDate(
+														timeFormat,
+														moment().startOf(
+															'day'
+														)
+													) }
+												</span>
+											),
+											endTime: (
+												<span>
+													{ formatDate(
+														timeFormat,
+														moment().endOf( 'day' )
+													) }
+												</span>
+											),
+											moreLink: (
+												<Link
+													href="https://woocommerce.com/document/managing-products/#product-data"
+													target="_blank"
+													type="external"
+													onClick={ () =>
+														recordEvent(
+															'add_product_learn_more',
```

Comment:
Does this name adhere to the convention of:
`<source>_<context>_<optional subcontext>_<action>_<optional qualifier>`?

Maybe we could also fix [this event name](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/products/sections/pricing-section.tsx#L111) as well.

Reply:
I used the existing `add_product_learn_more` event name. Let's address these in a followup PR.